src/Model/GalleryInterface.php

```diff
@@ -109,5 +109,10 @@ public function getGalleryHasMedias();
     /**
      * @param GalleryHasMediaInterface $galleryHasMedia
      */
-    public function addGalleryHasMedias(GalleryHasMediaInterface $galleryHasMedia);
+    public function addGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia);
+
+    /**
+     * @param GalleryHasMediaInterface $galleryHasMedia
+     */
+    public function removeGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia);
```

Comment:
won't this be a BC break? If someone is extending this interface?

Reply:
If you create a PR on master you will not see your changes until a very loooong time. Try making it BC. Creating a new interface and having gallery implement both is BC.